apps/server/src/modules/system/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './system.module';
-export * from './service';
+export { SystemService, LegacySystemService, SystemDto, OauthConfigDto, OidcConfigDto } from './service';
+export { System, SystemProps, OauthConfig, LdapConfig } from './domain';
```

Comment:
This is the correct place where we have to use explicit exports but imo we can do it everywhere like you already did :)